src/engine/renderer/glsl_source/vertexLighting_DBS_entity_fp.glsl

```diff
@@ -94,8 +94,8 @@ void	main()
 	// compute normal in world space from normalmap
 	vec3 N = NormalInWorldSpace(texCoords, tangentToWorldMatrix);
 
-	// compute the specular term
-#if defined(USE_REFLECTIVE_SPECULAR)
+	// compute the specular term, not compatible with PBR yet
+#if defined(USE_REFLECTIVE_SPECULAR) && !defined(USE_PHYSICAL_SHADING)
```

Comment:
I'm not sure exactly what it does but it might be interesting to look at defining `HasConflictingMacros` for this shader.

Reply:
hmmm, yes I forgot it, I just remember that it's very cryptic like all the macro code (and I'm even not sure that macro code works correctly at all) :confused: